makefile

```diff
@@ -41,10 +39,10 @@ $(buildDir)/protoc:
 	curl --retry 10 --retry-max-time 60 -L0 https://github.com/protocolbuffers/protobuf/releases/download/v$(protocVersion)/protoc-$(protocVersion)-$(protoOS).zip --output protoc.zip
 	unzip -q protoc.zip -d $(buildDir)/protoc
 	rm -f protoc.zip
-	GOBIN="$(abspath $(buildDir))" $(gobin) install github.com/golang/protobuf/protoc-gen-go@v$(protocGenGoVersion)
+	GOBIN="$(abspath $(buildDir))" $(gobin) install google.golang.org/protobuf/cmd/protoc-gen-go@latest
```

Comment:
Is it always okay to use the latest version of protoc-gen-go? Wouldn't that require us to also upgrade protoc to be sufficiently up-to-date for this to work?

Reply:
I think it should be fine, but we should probably be updating both either way. Unfortunately there is no "latest" URL for protoc.